src/main/java/module/specialEvents/SpecialEventsPanel.java

```diff
@@ -88,68 +64,54 @@ public void applicationClosing() {
 
-		TableColumn homeEventColumn = columnModel.getColumn(HOMEEVENTCOLUMN);
-		homeEventColumn.setMaxWidth(20);
-		homeEventColumn.setPreferredWidth(20);
-		homeEventColumn.setCellRenderer(new SETypeTableCellRenderer(false));
-
 		TableColumn homeTeamColumn = columnModel.getColumn(HOMETEAMCOLUMN);
-		homeTeamColumn.setPreferredWidth(150);
+		homeTeamColumn.setPreferredWidth(110);
 
 		TableColumn resultColumn = columnModel.getColumn(SpecialEventsTableModel.RESULTCOLUMN);
-		resultColumn.setPreferredWidth(40);
+		resultColumn.setPreferredWidth(30);
 
 		TableColumn awayTeamColumn = columnModel.getColumn(AWAYTEAMCOLUMN);
-		awayTeamColumn.setPreferredWidth(150);
-
-		TableColumn awayEventColumn = columnModel.getColumn(AWAYEVENTCOLUMN);
-		awayEventColumn.setMaxWidth(20);
-		awayEventColumn.setPreferredWidth(20);
-		awayEventColumn.setCellRenderer(new SETypeTableCellRenderer(true));
+		awayTeamColumn.setPreferredWidth(110);
 
 		TableColumn awayTacticColumn = columnModel.getColumn(AWAYTACTICCOLUMN);
-		awayTacticColumn.setPreferredWidth(37);
+		awayTacticColumn.setPreferredWidth(30);
 		awayTacticColumn.setCellRenderer(tacticsTableCellRenderer);
 
 		TableColumn minuteColumn = columnModel.getColumn(MINUTECOLUMN);
-		minuteColumn.setPreferredWidth(27);
-
-		TableColumn chanceColumn = columnModel.getColumn(CHANCECOLUMN);
-		chanceColumn.setMaxWidth(23);
-		chanceColumn.setPreferredWidth(23);
-		chanceColumn.setCellRenderer(new ChanceTableCellRenderer());
+		minuteColumn.setPreferredWidth(30);
+		minuteColumn.setMinWidth(30);
+		minuteColumn.setMaxWidth(30);
 
-		TableColumn eventTypeColumn = columnModel.getColumn(EVENTTYPCOLUMN);
-		eventTypeColumn.setMaxWidth(23);
-		eventTypeColumn.setPreferredWidth(23);
+		TableColumn eventTypeColumn = columnModel.getColumn(EVENTCOLUMN);
+		eventTypeColumn.setPreferredWidth(270);
+		eventTypeColumn.setMinWidth(200);
 		eventTypeColumn.setCellRenderer(new EventTypeTableCellRenderer());
 
-		TableColumn settExtColumn = columnModel.getColumn(SETEXTCOLUMN);
-		settExtColumn.setPreferredWidth(270);
-
-		TableColumn nameColumn = columnModel.getColumn(NAMECOLUMN);
-		nameColumn.setPreferredWidth(200);
+		TableColumn nameColumn = columnModel.getColumn(PLAYER_NAME_COLUMN);
+		nameColumn.setPreferredWidth(110);
 		nameColumn.setCellRenderer(new PlayerNameTableCellRenderer());
-		specialEventsTable.setRowHeight(20);
 
-		JSplitPane splitPane = new JSplitPane(JSplitPane.VERTICAL_SPLIT, filterPanel,
-				new JScrollPane(specialEventsTable));
+		specialEventsTable.addMouseListener(new MouseAdapter() {
+								  public void mousePressed(MouseEvent me) {
+									  JTable table = (JTable) me.getSource();
+									  Point p = me.getPoint();
+									  int col = table.columnAtPoint(p);
+									  if (col == MATCH_DATE_TYPE_COLUMN) {
+										  int matchID = 18113459; // TODO find how to get it for real
+										  HOMainFrame.instance().showMatch(matchID); // TODO only if match ID not null
```

Comment:
This should be easy to get from the model, something like (modulo all the null checks):

```java
SpecialEventsTableModel model = (SpecialEventsTableModel) table.getModel();
int matchID = model.getMatchRow(table.rowAtPoint(p)).getMatch().getMatchId();
```